functorch/experimental/_map.py

```diff
@@ -320,66 +356,25 @@ def map_fake_tensor_mode(mode, f, num_mapped, *args):
         return map_dense(f, num_mapped, *args)
 
 
-@map_impl.py_impl(DispatchKey.Functionalize)
-def map_func(f, num_mapped, *args):
-    reapply_views = torch._C._functionalization_reapply_views_tls()
+@map_impl.py_functionalize_impl()
```

Comment:
nit: Does py_functionalize_impl take in any args? If not then we should make it so that the () aren't necessary

Reply:
It does not - agreed (let me make sure that this works)